cli/checkpoint/checkpoint.go

```diff
@@ -45,7 +46,7 @@ func Cat(tntCli cmdcontext.TarantoolCli) error {
 	scanner := bufio.NewScanner(stdoutPipe)
 	scanner.Split(bufio.ScanLines)
 	for scanner.Scan() {
-		fmt.Println(scanner.Text())
+		fmt.Fprintf(os.Stderr, scanner.Text())
```

Comment:
The content must be printed to stdout. Only diagnostic messages must go to stderr.